src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubectlBuildWrapper.java

```diff
@@ -113,8 +120,10 @@ public void setUp(Context context, Run<?, ?> build, FilePath workspace, Launcher
             tlsConfig = " --insecure-skip-tls-verify=true";
         }
 
-        int status = launcher.launch()
-                .cmdAsSingleString("kubectl config --kubeconfig=\"" + configFile.getRemote()
+        kubectlBegin += "kubectl config --kubeconfig=\"";
+
+        int status = launcher.launch().envs("HTTPS_PROXY="+this.httpsProxy)
```

Comment:
Refactor `launcher.launch().envs("HTTPS_PROXY="+this.httpsProxy)` to a method to remove duplicates. Also, it should handle null `httpsProxy`.